spirekey/src/components/Registration/Registration.tsx

```diff
@@ -27,6 +26,11 @@ import Card from '../Card/Card';
 import CardBottom from '../Card/CardBottom';
 import DeviceIcons from '../Card/DeviceIcons';
 import PasskeyCard from '../Card/PasskeyCard';
+import {
+  CardContainer,
+  CardFooter,
+  SpireKeyCardContentBlock,
+} from '../CardPattern/CardPattern';
```

Comment:
I know this was already like this, but could you see if replacing the imports to use `@/components` work?